carbon-relay-ng/libtofs.jinja

```diff
@@ -0,0 +1,115 @@
+{%- macro files_switch(
```

Comment:
This file should be removed since TOFS isn't being used in this formula (the `files_switch` macro isn't imported anywhere).